packages/code-du-travail-frontend/src/annuaire/adresse.service.js

```diff
@@ -0,0 +1,55 @@
+import getConfig from "next/config";
+import memoizee from "memoizee";
+
+const {
+  publicRuntimeConfig: { API_URL }
+} = getConfig();
+
+function search(query) {
+  const url = `https://api-adresse.data.gouv.fr/search/?q=${encodeURIComponent(
```

Comment:
`https://api-adresse.data.gouv.fr` is a magic string that should come from a Magic contant file :)
Maybe from the `publicRuntimeConfig` like the `API_URL`
[Can be done in another PR]